internal/otelarrow/admission/boundedqueue.go

```diff
@@ -29,11 +33,12 @@ type waiter struct {
 	ID           uuid.UUID
 }
 
-func NewBoundedQueue(maxLimitBytes, maxLimitWaiters int64) *BoundedQueue {
+func NewBoundedQueue(tp trace.TracerProvider, maxLimitBytes, maxLimitWaiters int64) *BoundedQueue {
 	return &BoundedQueue{
 		maxLimitBytes:   maxLimitBytes,
 		maxLimitWaiters: maxLimitWaiters,
 		waiters:         orderedmap.New[uuid.UUID, waiter](),
+		tracer:          tp.Tracer("otel-arrow/admission"),
```

Comment:
@codeboten I tried to figure out how to have mdatagen spit out a generated_status.go file containing the ScopeName. I dug into the mdatagen sources and find that packages don't have scope name generated. What should I do?

Reply:
I've set the scope name to `github.com/open-telemetry/opentelemetry-collector-contrib/internal/otelarrow`